sympy/printing/tests/test_latex.py

```diff
@@ -1912,3 +1913,4 @@ def unimplemented_expr_sup_sub(expr):
     assert latex(unimplemented_expr(x)) == r'UnimplementedExpr\left(x\right)'
     assert latex(unimplemented_expr(x**2)) == r'UnimplementedExpr\left(x^{2}\right)'
     assert latex(unimplemented_expr_sup_sub(x)) == r'UnimplementedExpr^{1}_{x}\left(x\right)'
+    
```

Comment:
This blank line at the end of the file is causing the tests to fail.

Reply:
I've fixed it

Reply:
So this pull request can be merged now ?